docs/src/operator/installation.md

```diff
@@ -0,0 +1,85 @@
+# Installation
+
+We provide Debian packages for official releases for both the node software as well as a basic faucet implementation.
```

Comment:
The mentioning of the faucet comes out a bit "out of the blue". We should probably have a small section somewhere explaining what the faucet is and why it is needed.

Reply:
I've shuffled things around a bit, and I introduce the faucet in `# Architecture`.